auto_nag/scripts/close_intermittents.py

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bzcleaner import BzCleaner
+
+
+class Intermittents(BzCleaner):
+    def __init__(self):
+        super(Intermittents, self).__init__()
+
+    def description(self):
+        return "Intermittent test failure bugs unchanged in 21 days"
+
+    def get_bz_params(self, date):
```

Comment:
i) nit: remove the brackets they're useless
ii) the query is pretty hard to read like this: you should gather n,f,o,v by value:
   n4: 1,
   f4: ...
   o4: ...
   v4: ...
  it's pretty hard for my eyes to jump from a f to a o to v without missing the n.